### PR TITLE
docs: Add usage sub-section in docs

### PR DESCRIPTION
**Pull request by dreamorosi**
## Description of your changes

Following the changes introduced in #550 and [this comment](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/550#discussion_r815790620) from @saragerion, this PR introduces a new short "Usage" sub-section under the "Getting Started" section of each utility's page.

The goal of this section is to give a very basic example of the library's usage (just import & initialisation outside of handler) and point out why the utility has to be instantiated outside of the handler, explaining also benefits of doing so.

This wording was initially under the SAM section because that was the first code snippet but given that this _disclaimer_ is not SAM-specific it deserves its own section.

### Tracer
Screenshot 2022-02-28 at 16 06 05

### Logger
image

### Metrics
image

### Related issues, RFCs

[#550](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/550)  

### PR status

***Is this ready for review?:*** YES  
***Is it a breaking change?:*** NO

## Checklist

- [x] [My changes meet the tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
- [x] I have performed a *self-review* of my own code
- [x] I have made corresponding changes to the *documentation*
- [x] My changes generate *no new warnings*
- [x] The *code coverage* hasn't decreased
- [x] New and existing *unit tests pass* locally and in Github Actions
- [x] The PR title follows the [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/.github/semantic.yml#L2)

### Breaking change checklist

N/A

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by saragerion — APPROVED** [review]

File: docs/core/metrics.md [hunk 1]
```diff
@@ -43,6 +43,22 @@ Install the library in your project:
 npm install @aws-lambda-powertools/metrics
 ```
 
+### Usage
+
+The `Metrics` utility should always be instantiated outside of the Lambda handler. In doing this, subsequent invocations processed by the same instance of your function can reuse these resources. This saves cost by reducing function run time. In addition to this, `Metrics` can also be aware of things like whether or not a given invocation had a cold start or not and emit the appropriate metrics.
```
Comment (ijemmy) [inline docs/core/metrics.md:48]:
```suggestion
The `Metrics` utility must always be instantiated outside of the Lambda handler. In doing this, subsequent invocations processed by the same instance of your function can reuse these resources. This saves cost by reducing function run time. In addition, `Metrics` can track cold start and emit the appropriate metrics.
```

File: docs/core/logger.md [hunk 2]
```diff
@@ -29,6 +29,22 @@ Install the library in your project:
 npm install @aws-lambda-powertools/logger
 ```
 
+### Usage
+
+The `Logger` utility should always be instantiated outside of the Lambda handler. In doing this, subsequent invocations processed by the same instance of your function can reuse these resources. This saves cost by reducing function run time. In addition to this, `Logger` can also be aware of things like whether or not a given invocation had a cold start or not and inject the appropriate fields into the log.
```
Comment (ijemmy) [inline docs/core/logger.md:34]:
```suggestion
The `Logger` utility must always be instantiated outside of the Lambda handler. In doing this, subsequent invocations processed by the same instance of your function can reuse these resources. This saves cost by reducing function run time. In addition, `Logger` can track cold start and inject the appropriate fields into the logs.
```

File: docs/core/tracer.md [hunk 3]
```diff
@@ -31,6 +31,22 @@ Install the library in your project:
 npm install @aws-lambda-powertools/tracer
 ```
 
+### Usage
+
+The `Tracer` utility should always be instantiated outside of the Lambda handler. In doing this, subsequent invocations processed by the same instance of your function can reuse these resources. This saves cost by reducing function run time. In addition to this, `Tracer` can also be aware of things like whether or not a given invocation had a cold start or not and annotate your traces accordingly.
```
Comment (ijemmy) [inline docs/core/tracer.md:36]:
```suggestion
The `Tracer` utility must always be instantiated outside of the Lambda handler. In doing this, subsequent invocations processed by the same instance of your function can reuse these resources. This saves cost by reducing function run time. In addition, `Tracer` can track cold start and annotate the traces accordingly.
```

**Review by ijemmy — COMMENTED** [review]
I change the wording a bit to make it more accurate and concise.  What do you think?

**Review by ijemmy — APPROVED** [review]

**Review by saragerion — APPROVED** [review]

**Pull request merged**